org.eclipse.winery.workflowmodeler/src/app/app-routing.module.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (c) 2017 ZTE Corporation.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * and the Apache License 2.0 which both accompany this distribution,
+ * and are available at http://www.eclipse.org/legal/epl-v10.html
+ * and http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Contributors:
+ *     ZTE - initial API and implementation and/or initial documentation
+ */
+
+import {NgModule} from '@angular/core';
+import {RouterModule, Routes} from '@angular/router';
+import {WmToolbarComponent} from './components/toolbar/toolbar.component';
+
+const routes = [
+    {
```

Comment:
If we gonna have just one route, then we can actually think about to remove the AppRoutingModule and the dependency to "@angular/router". This would be just a nice-to-have change.